### PR TITLE
Project section changes

### PR DESCRIPTION
**Pull request by VasaviLagishetty**
projects
projects_mobile_view

### CODE REVIEW

File: _includes/projects.html [hunk 1]
```diff
@@ -3,51 +3,18 @@
...
     <div class="row">
-      <div class="col-lg-3 col-md-4 col-sm-6 mb-4">
-        <div class="card h-100">
-          <a href="#"><img class="card-img-top img-responsive" src="{{site.baseurl}}/img/projects/project.png" alt=""></a>
-          <div class="card-body">
-            <h4 class="card-title" style="margin-top: 10px;">
-              <a href="#">Project One</a>
-            </h4>
-            <p class="card-text">Lorem ipsum dolor sit amet, consectetur adipisicing elit. Amet numquam aspernatur eum quasi sapiente nesciunt? Voluptatibus sit, repellat sequi itaque deserunt, dolores in, nesciunt, illum tempora ex quae? Nihil, dolorem!</p>
+      {% for project in site.projects %}
```
Comment (SriSatyaLokesh) [inline _includes/projects.html:6]: good logic😁

**Review by SriSatyaLokesh — APPROVED** [review]

**Review by puneethkanna — COMMENTED** [review]
Everything is fine, but if a user give his project demo link, then will it be shown in the mobile visible screen of a user.

**Review by puneethkanna — APPROVED** [review]

**Comment by SriSatyaLokesh** [comment]
> Everything is fine, but if a user give his project demo link, then will it be shown in the mobile visible screen of a user.

We will plan this in future as in github we will have app link this is fine for now

**Comment by puneethkanna** [comment]
> > Everything is fine, but if a user give his project demo link, then will it be shown in the mobile visible screen of a user.
> 
> We will plan this in future as in github we will have app link this is fine for now

Okay sure!

**Review by SriSatyaLokesh — APPROVED** [review]

**Comment by VasaviLagishetty** [comment]
project
project_mobile

**Comment by SriSatyaLokesh** [comment]
That's good.
We can improve these in future
Let's merge this to master

**Comment by puneethkanna** [comment]
> That's good.
> We can improve these in future
> Let's merge this to master

We will think when user gives us..

**Pull request merged**